cinema_service/settings.py

```diff
@@ -20,12 +20,11 @@
 # See https://docs.djangoproject.com/en/4.0/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = (
-    "django-insecure-ru6ndfka@95_(lysua8yhdjq@vpiqgv3yru4r)q3h4_u8x7dfy"
-)
+SECRET_KEY = ("django-insecure-ru6ndfka@95_(l"
+              "ysua8yhdjq@vpiqgv3yru4r)q3h4_u8x7dfy")
```

Comment:
The `SECRET_KEY` should not be hardcoded in production settings. Consider using environment variables or a secrets manager to securely manage this key.